src/Support/OMOptions.cpp

```diff
@@ -45,3 +47,11 @@ llvm::cl::opt<std::string> acceleratorTarget("maccel",
     llvm::cl::desc("Specify an accelerator to generate code for\n"
                    "\"NONE\" or \"\" for no accelerator\n"),
     llvm::cl::init(""), llvm::cl::cat(OMPassOptions));
+
+llvm::cl::opt<OptLevel> OptimizationLevel(
```

Comment:
See my other comment. I think we should keep user facing options together and and use extern declarations to expose it to the accelerator code.  Options that are user facing should show in the --help output under the same category.